spec/features/managing_steps_spec.rb

```diff
@@ -181,4 +194,17 @@ def and_I_see_the_steps_updated_on_the_step_by_step_details_page
     expect(page).to have_css("table > tbody > tr:nth-child(1) > th:nth-child(2)", text: "Dress like the Fonz")
     expect(page).to have_css("table > tbody > tr:nth-child(2) > th:nth-child(2)", text: "Check how awesome you are")
   end
+
+  def and_I_write_a_change_note
+    fill_in "Description", with: "I've changed this step by step!"
+  end
+
+  def and_I_complete_a_change_note
+    and_I_write_a_change_note
```

Comment:
Not a deal breaker, but I would either move this step into the scenario or consolidate it into `and_I_complete_a_change_note`